scripts/go-to-slide.js

```diff
@@ -7,8 +7,10 @@ H5P.CoursePresentation.GoToSlide = (function ($) {
    * @class
    * @param {Number} slideNum
    * @param {CoursePresentation} cp
+   * @param {String} goToSlideNextPrevious
+   * @param {Number} index
    */
-  function GoToSlide(title, slideNum, invisible, cp) {
+  function GoToSlide(title, slideNum, invisible, cp, goToSlideNextPrevious, index) {
```

Comment:
_Re cp.currentSlideIndex_
You're right, I thought you could use it in the go function? I guess for the labels it is better to let them say "next slide", "previous slide"?

_Re the default values_
I had a look at the CP "constructor" and it didn't look the way I thought it would. It is ok the way you did it.